src/std/srfi/srfi-19.scm

```diff
@@ -261,14 +261,17 @@
 ;;; fare: added checking in make-time
 (defstruct time (type nanosecond second))
 
+(define (exact-integer? x)
+  (and (real? x) (exact? x) (integer? x)))
```

Comment:
(and (integer? x) (exact? x)) is denser

Reply:
Yes, but integer? is expensive, because 2.0 is an integer? so it has to do floating-point operations.

Reply:
ok.